modules/userId/eids.js

```diff
@@ -228,6 +228,10 @@ const USER_IDS_CONFIG = {
     source: 'amxrtb.com',
     atype: 1,
   },
+  'publinkId': {
+    source: 'epsilon.com',
```

Comment:
this seems incorrect; how is epsilon a source, they appear to be a consumer of the id